oktoberfest/spectral_library.py

```diff
@@ -148,7 +148,7 @@ def __init__(
 
     def gen_lib(self):
         """Read input csv file and add it to library."""
-        if self.config.fasta:
+        if self.config.library_input_type == "fasta":
```

Comment:
please add a check for a valid format here and add the functionality that only library input and library input type are taken from the config file.

Reply:
peptides.csv should be removed and in line 155, it should use the path to the file instead!